docs/intro.asciidoc

```diff
@@ -1,87 +1,10 @@
 [[intro]]
 == Introduction
 
-Centralized logging for .NET applications with the Elastic stack made easy.
-
-[role="screenshot"]
-image:https://user-images.githubusercontent.com/2163464/62682932-9cac3600-b9bd-11e9-9cc3-39e907280f8e.png[]
-
-[float]
-=== What is ECS?
-
-Elastic Common Schema (ECS) defines a common set of fields for ingesting data into Elasticsearch.
-For more information about ECS, visit the {ecs-ref}/ecs-reference.html[ECS Reference Documentation].
-
-[float]
-=== What is ECS logging?
-
-ECS integrations allow the use of Elastic Common Schema with your favorite logging library.
+ECS loggers are formatter/encoder plugins for your favorite logging libraries.
```

Comment:
```suggestion
ECS logging are integrations for your favorite .NET logging libraries.
```

Each logging library has its own named components e.g. formatters, layouts, etc. Maybe "integrations" is more agnostic?